packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -261,7 +272,7 @@ const NativeSelect = ({
         onBlur={handleOnBlur}
         ref={refCallback}
       >
-        {createPlaceholder(isFilterStyle, intl)}
+        {createPlaceholder(isFilterStyle, attrs, intl)}
```

Comment:
You can pass `updatedIsFilterStyle` here and remove the conditions added in `createPlaceholder` function.
```suggestion
        {createPlaceholder(updatedIsFilterStyle, intl)}
```